### DISCUSSION TITLE
Pharmacokinetic model in PKSIM for dogs

### DISCUSSION THREAD

**Discussion by CristianVU** [Open-Systems-Pharmacology/Forum#1108] [Models] [answered]
Hello.

I am a student and will start to develop a pharmacokinetic model in PK SIM for dogs.

I want to ask if there is an enzyme gene database for dogs or how it can be done to include the enzymes in dogs.

In the case of humans I have downloaded the base:
Gene expression database (human)

But for the case of the model in dogs I cannot use that.

From this moment I thank you for your answer.

**Comment by StephanSchaller** [comment, marked as answer]
Hi @CristianVU, this will be available shortly as a relative Expression Database. Please see here: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1096

Misssing this will be absolute reference concentrations. But maybe you can contribute here.

**Comment by CristianVU** [comment]
Thank you so much. Was there an update to the dog gene pool? Was there an official version released?
At the moment I am using the beta version posted on the forum.

**Reply by HenrikCordes** [reply]
Hi @CristianVU ,

no there is no public update yet. 

Did you had the chance to use the posted dog gene expression DB yet? 
Was it helpful for you? Did you found any bugs or had any complication using it, where all identifiers / gene names you where interested in available? 

It would be great if you could share you experience regarding https://github.com/Open-Systems-Pharmacology/Forum/discussions/1096 with the community or via personal communication. 

Best
Henrik
